sdk/language-support/java/codegen/src/it/java/com/daml/PackageNameAndVersionTest.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import tests.template1.SimpleTemplate;
+
+@RunWith(JUnitPlatform.class)
+public class PackageNameAndVersionTest {
+  @Test
+  void packageName() {
+    assertEquals(SimpleTemplate.PACKAGE_NAME, "integration-tests-model");
+  }
+
+  @Test
+  void packageVersion() {
+    assertEquals(SimpleTemplate.PACKAGE_VERSION, "1.2.3");
```

Comment:
consider if this should be a custom type that implements Comparable or whatever the Java interface is called. At least in CN we do rely on being able to compare versions. Ofc we can implement our own comparison but asking everyone to do so doesn't seem useful

Reply:
Very good point. I didn't consider it necessary atm but I'll look into it 👍